README.md

```diff
@@ -100,11 +100,16 @@ class MyComponent extends Component {
 }
 ```
 
-Options:
+In order to prevent the "Warning: Unknown props" message, you can use the `propsNamespace` option:
```

Comment:
Might be interesting to be a bit more didactic here. A link to the [React doc](https://facebook.github.io/react/warnings/unknown-prop.html) ? A statement about the fact that those props are not valid DOM attributes?

> The unknown-prop warning will fire if you attempt to render a DOM element with a prop that is not recognized by React as a legal DOM attribute/property.